### ISSUE TITLE
Missing label for dual-sim calls with Talkback

### ISSUE THREAD

**Issue by kujiu** [SimpleMobileTools/Simple-Dialer#720]
In the dialer in dual-SIM, I can't know which button is for SIM1 and which for SIM2 with the screen-reader.
